codex/stores/maintenance.nim

```diff
@@ -78,6 +78,7 @@ proc runBlockCheck(self: BlockMaintainer): Future[void] {.async.} =
     if be =? await maybeBeFuture:
       inc numberReceived
       await self.processBlockExpiration(be)
+      await sleepAsync(50.millis)
```

Comment:
Maybe we can do it as a configuration param?

Reply:
Or something we can remove once we get non-blocking IO in. It looks like this might just give time for the network async items to catch up? But yah, it'd be nice to have something a bit more concrete long term.